internal/bzlmod/semver.bzl

```diff
@@ -78,3 +78,28 @@ def _semver_to_comparable(v, *, relaxed = False):
 semver = struct(
     to_comparable = _semver_to_comparable,
 )
+
+def humanize_comparable_version(version):
```

Comment:
Let's export this as a field of the `semver` struct instead and call it "humanize_comparable".